endpoints-support.md

```diff
@@ -325,6 +325,14 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
 |<sub>/rest/scopes/resources/{resource_uri}</sub>                                         | GET      | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:| :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
 |<sub>/rest/scopes/resources/{resource_uri}</sub>                                         | PUT      | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:| :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
 |<sub>/rest/scopes/resources/{resource_uri}</sub>                                         | PATCH    | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:| :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+
+|     **Server Certificate**                                                                                                   |
```

Comment:
There are formatting errors here. Please correct it.

Reply:
@AnushaY1916 Corrected.